WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController/NotificationsViewController.swift

```diff
@@ -338,7 +342,11 @@ class NotificationsViewController: UIViewController, UIViewControllerRestoration
               let note = tableViewHandler.resultsController?.managedObject(atUnsafe: indexPath) as? Notification else {
             return UITableViewCell()
         }
-        cell.selectionStyle = .none
+        if splitViewControllerIsHorizontallyCompact {
+            cell.selectionStyle = .none
+        } else {
+            cell.selectionStyle = .default
```

Comment:
We are enabling it for all cases except for compact split view, correct?

Reply:
Yep. It's also disabled for the split mode on iPad:

https://github.com/wordpress-mobile/WordPress-iOS/assets/16563318/9b6f3710-3cb7-43d4-854e-1b8cc42d1ff3